common/changes/@microsoft/rush/dependency-analysis_2022-08-16-16-30.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Enriched the logged telemetry data by appending the build machine’s information e.g. number of cores, architecture, etc., individual project build times and status, and the dependency graph for the projects. An additional command line action `rush analyze` was added to do analysis on the enriched telemetry data. `rush analyze —dependency-analysis` and `rush analyze —simulate` were. —dependency-analysis uses the build times to identify which projects in the build are choke points where reducing the build time of these projects by 1 second will reduce the overall build time by 1 second. —simulate uses the build times to identify the optimal number of CPu cores to build your project with.",
```

```suggestion
      "comment": "Improved the logged telemetry data to include more details such as number of CPU cores, architecture, individual project build times and status, and the dependency graph for the projects",
```

This part should probably be a separate `rush change` entry:

> Added a new command "rush analyze-telemetry" which can identify projects with the largest impact on build times, as well as calculating the ideal amount of parallelism

The CHANGELOG.md entries are quick summaries, so we don't need to include too many details